fennel/datasets/datasets.py

```diff
@@ -986,7 +986,8 @@ def __init__(
         self.node.out_edges.append(self)
 
     def signature(self):
-        return fhash(self.node.signature(), self.columns)
+        # Doing this to make signature same as previous one.
```

Comment:
Do we use signature anywhere on the backend now? If not, we should probably just get rid of it someday. also cc @aditya-nambiar if he has any opinions.

Reply:
yeah have been planning to get rid of it for a while

Reply:
we use this in operator equality check just confirmed.